blinky-statik/src/main/java/org/spideruci/analysis/statik/Statik.java

```diff
@@ -25,6 +28,21 @@
   public static final String RTJAR = "/rt.jar";
   public static final String JCEJAR = "/jce.jar";
   
+  private static void RUN_SOOT(String[] args) {
```

Comment:
Yes I was thinking putting them in their own class and making the constructor private.